ui/src/views/Settings.vue

```diff
@@ -236,7 +236,7 @@ export default {
             console.error(e);
           }
         },
-        false
+        true; //true
```

Comment:
If the param is not passed, the default is `true` so you can delete the entire line

Reply:
you can check it here: https://github.com/NethServer/nethserver-cockpit/blob/master/root/usr/share/cockpit/nethserver/libs/nethserver.js#L3